src/ljudge.cc

```diff
@@ -108,6 +108,7 @@ namespace TestcaseResult {
   const string FLOAT_POINT_EXCEPTION = "FLOAT_POINT_EXCEPTION";
   const string SEGMENTATION_FAULT = "SEGMENTATION_FAULT";
   const string WRONG_ANSWER = "WRONG_ANSWER";
+  const string SKIPPED = "SKIPPED";
```

Comment:
Lines here are generated via [cog](https://nedbatchelder.com/code/cog/). I have updated the cog comment.